server/datastore/mysql/software.go

```diff
@@ -587,6 +587,7 @@ func listSoftwareDB(
 				cve.EPSSProbability = &result.EPSSProbability
 				cve.CISAKnownExploit = &result.CISAKnownExploit
 				cve.CVEPublished = &result.CVEPublished
+				cve.Description = &result.Description
```

Comment:
IIRC `opts.IncludeCVEScores` is only set for premium users: https://github.com/fleetdm/fleet/blob/d0b1e6b222a13773f4d24c3a289b2153e9cf406d/ee/server/service/hosts.go#L14-L16

(Let's check if CVE description should be a free/licensed feature.)

Reply:
And description does not belong to the scores IMO.

Reply:
Confirmed with @rachaelshaw that this is an EE only feature